### PR TITLE
Auto-label 'gemma' Issues/PRs. 

### PR DESCRIPTION
**Pull request by shmishra99**
This workflow automatically identifies issues and pull requests (PRs) related to `Gemma`.
It searches for the keyword `Gemma` (case-insensitive) in both the title and description of the issue/PR.
If a match is found, the workflow adds the label `Gemma` to the issue/PR.

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

Could we just have a single configurable auto-tagger that could be edited to add new tagging rules, instead of having to add a new set of files every time we want to add a new tag?

**Comment by shmishra99** [comment]
> Thanks for the PR!
> 
> Could we just have a single configurable auto-tagger that could be edited to add new tagging rules, instead of having to add a new set of files every time we want to add a new tag?

@fchollet , I've made the file generic so that next time we need auto-labeling, we can easily add sets of rules to the script file and additional trigger points within the same YAML file.

File: .github/workflows/scripts/labeler.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+
+/**
+ * Invoked from gemma-labeler.yaml file to add
```
Comment (fchollet) [inline .github/workflows/scripts/labeler.js:18]: Please update comment

File: .github/workflows/scripts/labeler.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+module.exports = async ({ github, context }) => {
+    const issue_title = context.payload.issue ?  context.payload.issue.title : context.payload.pull_request.title
+    const issue_discription = context.payload.issue ? context.payload.issue.body : context.payload.pull_request.body
+    const issue_number = context.payload.issue ? context.payload.issue.number : context.payload.pull_request.number
+    const labelToAdd = 'Gemma'
+    console.log(issue_title,issue_discription,issue_number)
+    if(issue_title.toLowerCase().indexOf('gemma') !=-1 || issue_discription.toLowerCase().indexOf('gemma') !=-1 ){
+        console.log(`Gemma keyword is present in #${issue_number} issue. Adding 'Gemma label.'`)
```
Comment (fchollet) [inline .github/workflows/scripts/labeler.js:31]: Can we abstract this to `keyword: label`, e.g. `"gemma": "Gemma"`?

e.g.:
```
config = {
   "gemma": "Gemma",
}
for keyword, label in config.items():
   if keyword in issue_title.lower() or keyword in issue_description.lower():
      add_label(label)
```

Comment (shmishra99) [reply .github/workflows/scripts/labeler.js]: Sure!! Created map to store keyword and label.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**